fe/fe-core/src/main/java/com/starrocks/sql/optimizer/base/HashDistributionSpec.java

```diff
@@ -117,7 +117,7 @@ public boolean isSatisfy(DistributionSpec spec) {
         }
 
         if (spec.type.equals(DistributionType.ROUND_ROBIN)) {
-            return true;
+            return false;
```

Comment:
why change it?

Reply:
some comment added,

ScanNode(LOCAL distribition property) satisfies required distribution property ROUND_ROBIN, so for union operator,  no PhysicalDistribution is enforced, it is unexpected.

Reply:
but this modify will let `ROUND_ROBIN` never return `true`, it's wilil take some cases failed?